akka-docs-dev/rst/scala/http/routing-dsl/directives/range-directives/withRangeSupport.rst

```diff
@@ -0,0 +1,48 @@
+.. _-withRangeSupport-:
+
+withRangeSupport
+================
+
+Transforms the response from its inner route into a ``206 Partial Content``
+response if the client requested only part of the resource with a ``Range`` header.
+
+
+Signature
+---------
+
+::
+
+    def withRangeSupport(): Directive0
+    def withRangeSupport(rangeCountLimit: Int, rangeCoalescingThreshold:Long): Directive0
+
+The signature shown is simplified, the real signature uses magnets. [1]_
+
+.. [1] See `The Magnet Pattern`_ for an explanation of magnet-based overloading.
+.. _`The Magnet Pattern`: /blog/2012-12-13-the-magnet-pattern/
```

Comment:
another relative path. Do a search for these, too, since searching for "spray" will not catch these.